### DISCUSSION TITLE
Firefox in Ubuntu 22.04.1 LTS

### DISCUSSION THREAD

**Discussion by lxxxvi** [actions/virtual-environments#6632] [General questions] [answered]
Hello 👋 

Is there a reason why Firefox is no longer available in Ubuntu 22.04.1 LTS?

**Comment by mikhailkoliada** [comment, marked as answer]
The reason was that canonical only let firefox be installed via a snap package, and only then added all the releases to its official PPA. We have recently added firefox to ubuntu 22.04 but this only will be released by the end of next week.
